content/en/guides/core/reports/edit-a-report.md

```diff
@@ -200,6 +200,183 @@ To freeze a run set when viewing a report, click the snowflake icon in its panel
 
 {{< img src="/images/reports/freeze_runset.png" alt="Freeze runset button" >}}
 
+## Filter a runset programmatically
+
+Programmatically filter run sets and add them to a report with the [Workspace and Reports API]({{< relref "/ref/python/wandb_workspaces/reports" >}}).
+
+The general syntax for a filter expression is:
+
+```text
+Filter('key') operation <value>
+```
+
+Where `key` is the name of the filter, `operation` is a comparison operator (e.g., `>`, `<`, `==`, `in`, `not in`, `or`, and `and`), and `<value>` is the value to compare against. `Filter` is a placeholder for the type of filter you want to apply. The following table lists the available filters and their descriptions:
```

Comment:
The details in line 213 seem duplicitative (but not a union set) with the table.